doc/rtd/topics/network-config-format-v1.rst

```diff
@@ -64,6 +64,11 @@ structure.
 The MAC Address is a device unique identifier that most Ethernet-based network
 devices possess.  Specifying a MAC Address is optional.
 
+.. note::
+
+  Whenever a MAC address consists of only the digits 0-9 (i.e. no hex a-f) it
```

Comment:
What do you think to being explicit about the exact requirement we have of MAC addresses, before going on to explain the way in which the reader can ensure they are meeting our requirement:

> MAC addresses must be strings. As MAC addresses which consist of only the digits 0-9 (i.e. no hex a-f) can be interpreted as a base 60 integer per [the YAML 1.1 spec](https://yaml.org/type/int.html), it is best practice to quote all MAC addresses to ensure they are parsed as strings regardless of value.

Reply:
I wasn't sure if the note should go into too much detail but have updated accordingly.